### PR TITLE
[DIA-273] Merge the loadPrivacyManager and loadOTTPrivacyManager functions

### PR DESCRIPTION
**Pull request by carmelo-iriti**

### CODE REVIEW

File: cmplibrary/src/main/java/com/sourcepoint/cmplibrary/SpConsentLibImpl.kt [hunk 1]
```diff
@@ -297,6 +297,8 @@ internal class SpConsentLibImpl(
 
         val gdprGroupPmId = campaignManager.getGroupId(campaignType)
 
+        pLogger.i("loadPm - isOtt: ", isOtt.toString())
```
Comment (Nevazhnovu) [inline cmplibrary/src/main/java/com/sourcepoint/cmplibrary/SpConsentLibImpl.kt:300]: Is this log desirable? Just asking 👀

File: cmplibrary/src/main/java/com/sourcepoint/cmplibrary/campaign/CampaignManager.kt [hunk 2]
```diff
@@ -262,6 +263,13 @@ private class CampaignManagerImpl(
             ?: false
     }
 
+    override fun isCampaignOtt(campaignType: CampaignType): Boolean {
+        return when (campaignType) {
+            CampaignType.GDPR -> dataStorage.isGdprOtt
+            CampaignType.CCPA -> false
```
Comment (Nevazhnovu) [inline cmplibrary/src/main/java/com/sourcepoint/cmplibrary/campaign/CampaignManager.kt:269]: Does it mean we don't have OTT for CCPA at all ?

File: cmplibrary/src/main/java/com/sourcepoint/cmplibrary/SpConsentLibImpl.kt (hunk 1 above)
Comment (carmelo-iriti) [reply cmplibrary/src/main/java/com/sourcepoint/cmplibrary/SpConsentLibImpl.kt:300]: yes, pLogger in prod is an empty object, in debug allows us to test if the `isOtt ` has the right value

File: cmplibrary/src/main/java/com/sourcepoint/cmplibrary/campaign/CampaignManager.kt (hunk 2 above)
Comment (Nevazhnovu) [reply cmplibrary/src/main/java/com/sourcepoint/cmplibrary/campaign/CampaignManager.kt:269]: I'm not aware either

**Pull request merged**